visidata/loaders/postgres.py

```diff
@@ -115,17 +162,19 @@ def reload(self):
                 self.addRow(r)
 
     def openRow(self, row):
-        return PgTable(self.name+"."+row[0], source=row[0], sql=self.sql)
+        return PgTable(
+            self.name+"."+row[0],
+            source=(self.source or options.postgres_schema)+"."+row[0],
```

Comment:
Seems like the given name and the source should be the same here.  Let's use `source`, since I think it's reasonable to require `source` to be the schema for this class, and only use `options.postgres_schema` in the outer `openurl_postgres` function.